Google.Ads.GoogleAds/examples/AdvancedOperations/AddAdCustomizer.cs

```diff
@@ -126,337 +113,190 @@ public void Run(GoogleAdsClient client, long customerId, long[] adGroupIds)
-            string feedName)
+        private string CreateTextCustomizerAttribute(GoogleAdsClient client, long customerId,
+            string customizerName)
         {
-            // Get the FeedServiceClient.
-            FeedServiceClient feedService = client.GetService(Services.V11.FeedService);
-
-            // Creates three feed attributes: a name, a price and a date. The attribute names
-            // are arbitrary choices and will be used as placeholders in the ad text fields.
-            FeedAttribute nameAttribute = new FeedAttribute()
-            {
-                Name = "Name",
-                Type = FeedAttributeType.String
-            };
-
-            FeedAttribute priceAttribute = new FeedAttribute()
-            {
-                Name = "Price",
-                Type = FeedAttributeType.String
-            };
+            CustomizerAttributeServiceClient customizerAttributeService = 
+                client.GetService(Services.V11.CustomizerAttributeService);
 
-            FeedAttribute dateAttribute = new FeedAttribute()
+            CustomizerAttribute nameAttribute = new CustomizerAttribute()
             {
-                Name = "Date",
-                Type = FeedAttributeType.DateTime
+                Name = customizerName,
+                Type = CustomizerAttributeType.Text
             };
 
-            Feed adCustomizerFeed = new Feed()
-            {
-                Name = feedName,
-                Attributes = { nameAttribute, priceAttribute, dateAttribute }
+            CustomizerAttributeOperation nameAttributeOperation = new CustomizerAttributeOperation(){
+                Create = nameAttribute
             };
 
-            FeedOperation feedOperation = new FeedOperation()
-            {
-                Create = adCustomizerFeed
-            };
+            MutateCustomizerAttributesResponse response =
+                customizerAttributeService.MutateCustomizerAttributes(customerId.ToString(),
+                    new[] {nameAttributeOperation});
 
-            MutateFeedsResponse response =
-                feedService.MutateFeeds(customerId.ToString(), new[] { feedOperation });
+            string customizerAttributeResourceName = response.Results[0].ResourceName; 
+            Console.WriteLine($"Added name customizer attribute with resource name" +
+                $" '{customizerAttributeResourceName}'.");
 
-            string feedResourceName = response.Results[0].ResourceName;
-            Console.WriteLine($"Added feed with resource name '{feedResourceName}'.");
-            return feedResourceName;
+            return customizerAttributeResourceName;
         }
         // [END add_ad_customizer]
 
         /// <summary>
-        ///  Retrieves all the attributes for a feed and returns them in a map using the
-        ///  attribute names as keys.
+        /// Creates a price customizer attribute ad returns its resource name.
```

Comment:
Many of the comments on `CreateTextCustomizerAttribute` apply here as well.

Reply:
Done.